data-prepper-core/src/main/java/org/opensearch/dataprepper/peerforwarder/server/NoOpPeerForwarderServer.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.peerforwarder.server;
+
+/**
+ * Class to handle NoOp Peer Forwarder server if no peers are configured
+ *
+ * @since 2.0
+ */
+public class NoOpPeerForwarderServer implements PeerForwarderServer {
```

Comment:
I prefer testing no-op methods. It can be as simple as calling them to ensure that they run.